src/app/modules/rup/components/elementos/checklist.html

```diff
@@ -0,0 +1,28 @@
+<form #form="ngForm">
+    <plex-list size="sm" *ngIf="!soloValores">
```

Comment:
Algunas sugerencias:
- El `plex-bool` cuenta con el input 'label', nos podemos ahorrar abrir otro tag.
- Evaluar si es pertinente el uso de un plex-item, quizás se podría iterar directamente sobre el `<plex-bool>` y envolver todo en un `<plex-grid>` de modo que ocupe menos espacio, paso a graficar:

`<plex-grid size="md" type="full" *ngIf="!soloValores">`
` <plex-bool *ngFor="let concepto of conceptos ; let i =index" label="{{ concepto.term }}" type="checkbox" [name]="'opcion'+i" [(ngModel)]="concepto.value" (change)="selectCheck(concepto)">`
`<plex-bool>`
`</plex-grid>`

Reply:
@JSantarelli cambios sugeridos aplicados.